src/types.js

```diff
@@ -3,3 +3,54 @@ export type Data = {
   key: string,
   value: string,
 };
+
+export type GlobalAttributes = $Shape<{|
+  hidden: boolean,
+  id: string,
+  style: { [key: string]: any },
+  tabIndex: string | number,
+  title: string,
+|}>;
+
+// Event handlers
+
+export type SyntheticMouseEventHandler<T: EventTarget = EventTarget> =
```

Comment:
these look good; we can then use them in SS as well. I'd like to see how/where exactly we could use them in SS and which advantages do they give us? will this fix `event.currentTarget`, etc. issues?

Reply:
It should, yes. I do think it could be helpful.
We should also maybe add an ESLint warning against using `evt.target` as well.